components/DetailCountry/index.js

```diff
@@ -15,10 +15,11 @@ export default function DetailCountry({
   newImg,
   addImage,
```

Comment:
```suggestion
  newImg,
  addImage,
```
that seems like doppelt gemoppelt. I can't see all the code related in this PR, but make sure these don't do the same thing and if no, that the naming is a bit clearer :)